cupy_backends/hip/cupy_profiler.h

```diff
@@ -11,6 +12,14 @@ hipError_t hipProfilerInitialize(...) {
     return hipSuccess;
 }
 
+void hipProfilerStart() {
```

Comment:
@pnunna93 This might fail because, the profiler API will have ```hipError_t hipProfilerStart()``` as the signature. Use the above signature and call the function ```roctracer_start()``` inside this method and return a hipSuccess. This will also avoid adding an if condition for the User APIs in profiler.pyx functions. 
On a side note, Is there a plan to use roctracer_start() instead of hipProfilerStart inside hipify_torch?

Reply:
Yes, I have opened a PR to replace hipProfilerStart/Stop with roctracer functions:
https://github.com/ROCmSoftwarePlatform/hipify_torch/pull/60/files

Reply:
Above PR is merged into hipify_torch, so I have removed hipProfilerStart/Stop from cupy_backends/hip/cupy_profiler.h instead.